src/parser/jobs/whm/index.js

```diff
@@ -26,23 +25,35 @@ export default new Meta({
 				<Trans id="whm.about.description.devwarning">The WHM module is still in a very early developement stage.  If you notice any issues, or have any concerns, or suggestions on what you would like this module to analyze next, please drop by our Discord channel!</Trans>
 			</Message.Content>
 		</Message>
-		<Trans id="whm.about.description.support">Currently this module can track DoT uptime, detect clipping and report about missed Divine Benison and Assize casts.</Trans>
-
+		<Trans id="whm.about.description.support">Currently this module can track DoT uptime, Lily and Blood Lily Usage, and detect clipping and report about missed Divine Benison and Assize casts.</Trans>
 		<Message warning icon>
 			<Icon name="warning sign"/>
 			<Message.Content>
 				<Trans id="whm.about.description.invulnwarning">Currently, for all healers, boss invulnerability checking doesn't function properly. This results in inaccuracy for many time-related functions such as the Always be casting checklist item or DoT uptime.</Trans>
 			</Message.Content>
 		</Message>
 	</>,
-	// supportedPatches: {
-	// 	from: '4.05',
-	// 	to: '4.5',
-	// },
+	supportedPatches: {
```

Comment:
As supported as it can be once I get through your changes.  WHM hasn't changed a lot from 4.5 except the addition of the lilies, and I've added lilies to our tracking and updated everything else.  I absolutely would like to consider ways to deepen the support, but the important stuff is all here.